libraries/botframework-connector/src/auth/jwtTokenValidation.ts

```diff
@@ -5,19 +5,19 @@
  * Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Changes here are primarily code formatting and moving non-exported functions outside the namespace.